packages/turbo-repository/npm/darwin-arm64/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@turbo/repository-darwin-arm64",
```

Comment:
how does these platform-speciifc packages get built and published for `turbo`? Could we eventually match that and not keep each of these in source control?

Reply:
We could standardize at some point. We have a [script](https://github.com/vercel/turbo/blob/main/cli/scripts/npm-native-packages/npm-native-packages.js) for `turbo`, the method here is copied from [next-swc](https://github.com/vercel/next.js/tree/canary/packages/next-swc/crates/napi/npm) in the Next.js repo.